electrumx/lib/coins.py

```diff
@@ -51,7 +51,13 @@
                                       SmartCashElectrumX, AuxPoWElectrumX)
 
 
-Block = namedtuple("Block", "raw header transactions")
+@dataclass
+class Block:
+    __slots__ = "raw", "header", "transactions"
+    raw: ByteString
+    header: ByteString
```

Comment:
What are the actual types used here? Is these not just `bytes`?
https://github.com/spesmilo/electrumx/blob/7e1a0288537c1cf1d202fbadff3a7f0f93b7565e/electrumx/server/daemon.py#L229
I've never used use `ByteString` to annotate before, just looking at https://github.com/python/mypy/issues/4871
but if you think it fits; fine by me.

Reply:
It was a placeholder until I could confirm we weren't constructing a bytearray.